### DISCUSSION TITLE
Is excimer extension supported or not?

### DISCUSSION THREAD

**Discussion by lunfel** [mlocati/docker-php-extension-installer#789] [Q&A]
I am trying to install excimer through docker-php-ext-install, but I am getting an error that it is not found even if in the README I can find the excimer extension listed. I am using php7.4 and php8.2.

For master branch:
![image](https://github.com/mlocati/docker-php-extension-installer/assets/1844873/a6487e2c-7e89-41c4-81a2-3f4e33bc4630)

I can see a few PR that relates to adding or fixing things about excimer (in chronological order):
- https://github.com/mlocati/docker-php-extension-installer/pull/309/files
- https://github.com/mlocati/docker-php-extension-installer/pull/570/files,
- (remove) https://github.com/mlocati/docker-php-extension-installer/pull/571/files
- https://github.com/mlocati/docker-php-extension-installer/pull/714/files
- (remove) https://github.com/mlocati/docker-php-extension-installer/pull/720/files

The last PR is named "Install newer excimer on PHP 7.1-7.3 too" but it only removes code related to excimer. Is this an error?

If I sift through https://github.com/mlocati/docker-php-extension-installer/blob/master/install-php-extensions on master branch, there is no reference to excimer.

What is happening here? Was it mistakenly removed?

**Comment by mlocati** [comment]
It's supported.
Do you have any issue installing it?

**Comment by lunfel** [comment]
Oh this is me being dumb. I misread the official documentation from docker php. 

"if you are having difficulty figuring out which Debian or Alpine packages need to be installed before docker-php-ext-install, then have a look at the install-php-extensions project." which points to this repository. I thought this the `docker-php-ext-install` script in the official docker image was managed in this repository. I was trying to install excimer through `docker-php-ext-install` and not through `install-php-extensions`.

But then I missed that line: "This tool is contributed by community members and is not included in the images, please refer to their Git repository for installation, usage, and issues."

I followed the instructions to install `install-php-extensions` and then install excimer and everything was fine.

Sorry about the confusion.

**Comment by mlocati** [comment]
No problem! I know that the script names may be confusing. 
The important thing is that you managed to do what you need to do!
